packages/components/src/components/common/table/footer/footer.tsx

```diff
@@ -16,6 +25,7 @@ const TableAddRow = ({ addRow, tabindex }: { addRow: () => void; tabindex?: numb
         variant='outline'
         aria-label='Add row'
         tabIndex={tabindex}
+        onKeyDown={onKeyDown}

```

Comment:
why needed?

Reply:
you could also imply allow all possible button props instead of add all one by one...